### PR TITLE
fix/rename segmentation namespace

### PR DESCRIPTION
**Pull request by satoshi-ota**
## PR Type

- rename segmentation directory
- update topic namespace /perception/object_segmentation/pointcloud -> /perception/obstacle_segmentation/pointcloud

Related PR

- https://github.com/tier4/AutowareArchitectureProposal.iv/pull/742
- https://github.com/tier4/AutowareArchitectureProposal.iv/pull/739
- https://github.com/tier4/AutowareArchitectureProposal_aip_launcher/pull/6

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: planning_launch/launch/scenario_planning/parking.launch.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def generate_launch_description():
                     ("~/input/objects", "/perception/object_recognition/objects"),
                     (
                         "~/input/points_no_ground",
```
Comment (yukkysaito) [inline planning_launch/launch/scenario_planning/parking.launch.py:56]: 今回のPRと分けても大丈夫ですが、時間があればno_groundというワードは使わないように変更したいです。

**Review by yukkysaito — APPROVED** [review]

File: planning_launch/launch/scenario_planning/parking.launch.py (hunk 1 above)
Comment (satoshi-ota) [reply planning_launch/launch/scenario_planning/parking.launch.py:56]: I'll fix it in other PRs :+1:

Comment (yukkysaito) [reply planning_launch/launch/scenario_planning/parking.launch.py:56]: Okay :+1:

**Comment by satoshi-ota** [comment]
I'm resolving confilicts now...

**Pull request merged**